### PR TITLE
Home Component & Route Updates

### PR DESCRIPTION
**Pull request by GracieEveritt**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,11 +15,11 @@ function App() {
...
-            <Route exact path="/" component={Login} />
-            <Route path="/login" component={Login} /> 
+            <Route exact path="/" component={Home} />
+            <Route path="/dashboard" render={routerProps => <Dashboard match={routerProps}/>}/>
+            <Route path="/login" component={Login} />
             <Route path="/signup" component={Signup} />
-            <Route path="/:id" render={routerProps => <Dashboard match={routerProps}/>}/>
-            <Redirect to="/login"/>
+            <Redirect to="/"/>
         </Switch>
```
Comment (bcantello) [inline src/App.js:23]: Let's move the routing into a Main component

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import SignUp from './SignUp'
+
+function Home() {
+    
+
```
Comment (bcantello) [inline src/components/Home.js:6]: nit: I know we will add things here eventually, but lets get rid of the extra white space for now

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    
+    return(
+        <>
+        <div>Home</div>
+        <SignUp />
+        </>
+    )
+}
+
+export default Home
```
Comment (bcantello) [inline src/components/Home.js:15]: nit: add line of white space at the end of the file

File: src/App.js (hunk 1 above)
Comment (GracieEveritt) [reply src/App.js:23]: K - will do on Monday.

**Pull request merged**